core/client/fs/src/main/java/alluxio/client/file/cache/PageStore.java

```diff
@@ -171,27 +183,16 @@ ReadableByteChannel get(PageId pageId, int pageOffset) throws IOException,
   void delete(PageId pageId, long pageSize) throws IOException, PageNotFoundException;
 
   /**
-   * @return the number of pages stored
-   */
-  long pages();
-
-  /**
-   * @return the total size of pages stored in bytes
-   */
-  long bytes();
-
-  /**
-   * Gets all page ids.
+   * Gets a stream of all pages from the page store. This stream needs to be closed as it may
```

Comment:
(nit) Given all other stream usages do not require explicit closing, it might be hard for people to remember to close it.

Reply:
perhaps. but we should either use iterator or stream here because we will need to handle ~1 million pages restored, that could take ~GB memory if we use collection. due to this reason, I and @calvinjia  after discussion decided to stick to stream.